playwright/playwright.config.ts

```diff
@@ -1,7 +1,46 @@
 import { type PlaywrightTestConfig, devices } from "@playwright/test";
+import assert from "assert";
+import dotenv from "dotenv";
+
+dotenv.config({ path: ".env.test" });
+dotenv.config({ path: ".env.test.local", override: true });
+
+declare const process: {
+  env: {
+    CI: boolean;
+    PLAYWRIGHT_ADMIN_USERNAME: string;
+    PLAYWRIGHT_ADMIN_PASSWORD: string;
+    PRISMIC_CLUSTER?: string;
+    MANAGEV2_SECRET: string;
+    MANAGEV2_AUDIENCE: string;
+    BASE_URL: string;
+    E2E_REPOSITORY: string | undefined;
+  };
+};
+
+export const baseUrl = process.env.BASE_URL;
+
+export const prismicCluster = process.env.PRISMIC_CLUSTER;
+
+export const auth = {
```

Comment:
Same users as in https://github.com/prismicio/editor/blob/main/packages/wroom/e2e/globalSetup.ts